dist/index.js

```diff
@@ -21988,9 +21988,28 @@ async function install(version2, chmod) {
   return binaryPath;
 }
 function downloadURL(version2) {
+  version2 = version2.startsWith("v") ? version2.slice(1) : version2;
+  const isHigherVersion = isVersionGreaterThan371(version2);
   const extension = process.platform === "win32" ? "exe" : process.platform;
+  if (isHigherVersion && extension !== "exe") {
+    let arch = process.arch.toString();
+    if (arch === "x64") {
+      arch = "amd64";
+    }
+    return `https://github.com/getsops/sops/releases/download/v${version2}/sops-v${version2}.${extension}.${arch}`;
+  }
   return `https://github.com/getsops/sops/releases/download/v${version2}/sops-v${version2}.${extension}`;
 }
+function isVersionGreaterThan371(version2) {
+  const versionParts = version2.split(".");
+  const major = parseInt(versionParts[0]);
+  const minor = parseInt(versionParts[1]);
+  const patch = parseInt(versionParts[2]);
```

Comment:
can be simplified afaik
```suggestion
  const [major, minor, patch] = version2.split(".").map(v => parseInt(v, 10));
```